Terminal.Gui/Views/Tab.cs

```diff
@@ -1,39 +1,42 @@
 ﻿namespace Terminal.Gui; 
 
 /// <summary>
-/// A single tab in a <see cref="TabView"/>
+/// A single tab in a <see cref="TabView"/>.
 /// </summary>
-public class Tab {
-	private string text;
+public class Tab : View {
+	private string _displayText;
 
 	/// <summary>
-	/// The text to display in a <see cref="TabView"/>
+	/// The text to display in a <see cref="TabView"/>.
 	/// </summary>
 	/// <value></value>
-	public string Text { get => text ?? "Unamed"; set => text = value; }
+	public string DisplayText { get => _displayText ?? "Unamed"; set => _displayText = value; }
```

Comment:
I still need the `DisplayText` property to set the real text even it's bigger to display. I'll try to use `Title` instead of `Text` and see what is happen.

Reply:
You're not getting my point. 

But if you just want to move on, it's ok as this is something that can be refined later. What you've done with this so far is really great and what I'm asking for is probably overkill.

Feel free to resolve all the rest of my comments and I'll merge.

Separately, when I get a chance I'll do a write up further explaining myself.